GitHub Issues Service Test Failing

:clock11: **When did the problem start?**
At least ~10 days prior to the opening of this issue (~Jan 15th). It was likely failing for some time before, but was lost in the see of the full service test failures (1900+) that we had for a while 

<!-- Indicate when the problem started -->

:camera: **Live badge**
https://img.shields.io/github/issues-raw/badges/shields/service-badge
<!-- Provide a link to the live badge in plain text and markdown.  -->

![](https://img.shields.io/github/issues-raw/badges/shields/service-badge)

:wrench: **Is the live badge working?**
Yes, seems to be
<!-- Indicate whether or not the live badge is working. -->

:link: **CircleCI link**
https://app.circleci.com/pipelines/github/badges/daily-tests/734/workflows/0288e2f4-3f93-4662-97a8-7876dbc1c757/jobs/1632
<!-- Provide a link to the failing test in CircleCI. -->

:beetle: **Stack trace**

```
GithubIssues [live] GitHub open issues by label including slash character (raw) [ GET /issues-raw/IgorNovozhilov/ndk/@ndk/cfg.json ] - [ GET /issues-raw/IgorNovozhilov/ndk/@ndk/cfg.json ]
/home/circleci/project/shields/core/service-test-runner/cli.js

AssertionError: label mismatch: expected 'issues' to equal 'open @ndk/cfg issues'
    at Function._expectField (core/service-test-runner/icedfrisby-shields.js:69:51)
    at IcedFrisbyNock.<anonymous> (core/service-test-runner/icedfrisby-shields.js:53:26)
    at IcedFrisbyNock.<anonymous> (node_modules/icedfrisby/lib/icedfrisby.js:954:10)
    at invokeNextHook (node_modules/icedfrisby/lib/icedfrisby.js:1003:24)
    at /home/circleci/project/shields/node_modules/icedfrisby/lib/icedfrisby.js:1017:7
    at new Promise (<anonymous>)
    at IcedFrisbyNock._runHooks (node_modules/icedfrisby/lib/icedfrisby.js:976:12)
    at IcedFrisbyNock.run (node_modules/icedfrisby/lib/icedfrisby.js:1276:20)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async Context.<anonymous> (node_modules/icedfrisby/lib/icedfrisby.js:1348:9)
```

:bulb: **Possible solution**
Not sure, possible the target repo was modified/removed/etc.
<!--- Optional: only if you have suggestions on a fix/reason for the bug -->

<!-- Love Shields? Please consider donating $10 to sustain our activities:
👉  https://opencollective.com/shields -->

--- a/services/github/github-issues.tester.js
+++ b/services/github/github-issues.tester.js
@@ -82,9 +82,9 @@ t.create('GitHub open issues by label (raw)')
 
 // https://github.com/badges/shields/issues/1870
 t.create('GitHub open issues by label including slash character (raw)')
-  .get('/issues-raw/IgorNovozhilov/ndk/@ndk/cfg.json')
+  .get('/issues-raw/calebcartwright/shields-service-test-target/@foo/bar.json')
   .expectBadge({
-    label: 'open @ndk/cfg issues',
+    label: 'open @foo/bar issues',
     // Not always > 0.
     message: Joi.alternatives(isMetric, Joi.equal('0')),
   })

